### ISSUE TITLE
Prior distribution "FIX" for remnant error 

### ISSUE THREAD

**Issue by Felipemendezrios** [BaM-tools/BaM#8]
It appears that using a prior distribution of type "FIX" doesn't work for the remnant error model. However, the same approach has been successfully employed in the config_model.

**Comment by benRenard** [comment]
Indeed, and it's problematic because BaRatinAGE relies on FIX parameters to specify constant or purely proportional error models.

Error message is as follows:  
Adaptive_Metro_OAAT:Posterior_wrapper:GetPostLogPdf:GetPriorLogPdf:GetLogPrior:FATAL:[PriorList(i)%par] is not allocated

**Comment by benRenard** [comment]
In fact, none of FIX, VAR or STOK parameter types are allowed for remnant error parameters. Supporting this would require some major restructuring of the code, and I don't think it's worth it - maybe it will be worth it in the future when this part of the code will be rewritten to support e.g. spatial error models.

In the meantime, the following was implemented in commit [5f764f9](https://github.com/BaM-tools/BaM/commit/5f764f9041f1ebdb7d2479ec1b0e42aad9bda53d):
1. Explicit error message to indicate that FIX, STOK or VAR parameters are not supported for remnant error parameters 
2. A new sigma function was implemented: "Proportional": sigma = gamma*abs(Y)
